audit/audit.go

```diff
@@ -113,6 +125,8 @@ func NewPostgresAuditor(connStr string) (*PostgresAuditor, error) {
 	return pg, nil
 }
 
+// Handle is the PostgresAuditor implementation of the function required by the
```

Comment:
Let's describe this function (at a high-level) rather than just describing it as "required by the interface"

Reply:
I think it should be described at a high level on the interface and not on this particular implementation. We might want to document some specifics to postgres here, though.

Reply:
+1 @autrilla

Reply:
All PR comments have been addressed except for this one. I will let you two decide what to write :)

Reply:
@autrilla agreed

@adrianosela 
on the interface, append something like: "Handle() takes an audit event and attempts to persists it. How it is persisted and how errors are handled is up to the implementation of this interface."

on `PostgresAuditor.Handle()`, something like: "Persists the audit event by writing a row to the 'audit_event' table. Errors with writing to the database will output a log message and the process will exit with status set to 1."